### ISSUE TITLE
Package Vosk for Debian

### ISSUE THREAD

**Issue by besentv** [alphacep/vosk-api#1308]
Currently, Vosk seems only packaged for PIP, but I'd like to use it as a dependency in Wine which expects the library to be installed as a system package (as well as its CI). So, packaging libvosk.so and vosk-api.h for Debian would be great.

**Comment by nshmyrev** [comment]
Yes, it would be nice to have the package. But I suppose packaging will not be simple since we require custom openfst and kaldi too

**Comment by besentv** [comment]
Doesn't Vosk packaged for pip link against these statically? (Meaning dependencies wouldn't be an issue.)

**Comment by nshmyrev** [comment]
Yes, but I suppose debian approach would be to package everything separately. Yet to think on this. I hope we will move soon to pytorch, it will make packaging easier.

**Comment by besentv** [comment]
As long as the C interface stays the same I'm happy. 😄

**Comment by johngebbie** [comment]
I don't know the Debian packaging system but you might find the template [here](https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/testing/vosk-api) for Alpine helpful. It has patches to use the latest versions of openfst and openblas.

**Comment by hackerb9** [comment]
Is the move to pytorch still planned?

**Comment by nshmyrev** [comment]
> Is the move to pytorch still planned?

Yes, you can use sherpa-onnx meanwhile
